.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
 max-line-length = 120
-exclude = build, venv
+exclude = build, venv, .tox
```

Comment:
what is .tox ?

Reply:
It's a cache for the tool `tox`. tox something use to test our lib against a matrix of dependencies and Python versions